UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common/Helpers/FileHelper.cs

```diff
@@ -3,16 +3,17 @@
     public static class FileHelper
     {
         private static readonly string[] suffixes = { "Bytes", "KB", "MB", "GB", "TB", "PB" };
+
         public static string FormatSize(long bytes)
         {
             int counter = 0;
             decimal number = (decimal)bytes;
-            while (Math.Round(number / 1024) >= 1)
+            while ((number / 1024) >= 1)
```

Comment:
Parentheses redundant:
```suggestion
            while (number / 1024 >= 1)
```

Reply:
Removed parentheses